src/lib/i18n-data.js

```diff
@@ -14,6 +15,7 @@ export default {
   "fr-ca": fr,
   "fr-us": fr,
   "ht-us": ht,
+  "ja": jajp,
```

Comment:
Should this be reversed? `"ja-jp": ja` The rest of the locales have the dash and the rest of the variables are just 2 letters unless there is a conflict